modules/collect/src/main/java/com/opengamma/strata/collect/MapStream.java

```diff
@@ -58,7 +58,7 @@
    * applying a function to each value.
    *
    * @param collection  the collection of values
-   * @param keyFunction  a function that returns the key for a value
+   * @param keyFunction  a function which returns the key for a value
```

Comment:
This claims both versions are correct

http://www.oxforddictionaries.com/words/that-or-which